modules/backup-etcd.adoc

```diff
@@ -25,4 +23,12 @@ You should only save a backup from a single master host. You do not need a backu
 $ sudo /usr/local/bin/etcd-snapshot-backup.sh ./assets/backup
 ----
 +
-In this example, the etcd snapshot and static Kubernetes API server resources are saved to `./assets/backup/snapshot_db_kuberesources_<datetimestamp>.tar.gz` on the master host.
+In this example, two files are created in the `./assets/backup/` directory on the master host:
+
+* `snapshot_<datetimestamp>.db`: The etcd snapshot
+* `static_kuberesources_<datetimestamp>.tar.gz`: The static Kubernetes API server resources
++
+[NOTE]
+====
+If etcd encryption is enabled, this file contains the encryption keys for the etcd snapshot. In this case, it is recommended to store this file separately from the etcd snapshot for security reasons. However, this file is required in order to restore from the etcd snapshot.
```

Comment:
better: "the second file". Otherwise, it's not clear which is meant.

Reply:
Rearranged wording to hopefully be clearer.